### PR TITLE
API URL Changed And code formating

### PR DESCRIPTION
**Pull request by anupkore**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -41,6 +44,7 @@ import ProtectedRoute from './Components/ProtectedRoute';
 
 
 
+
```
Comment (Techvarun1991) [inline src/App.js:47]: Remove Extra Spaces

File: src/Components/AddNewMember.js [hunk 2]
```diff
@@ -23,6 +23,12 @@ export default function AddNewMember()
       return emailRegex.test(email);
     }
 
+    // function validateContact(contact) 
+    // {
+    //   const contactRegex = /^\d$/;
+    //   return contactRegex.test(contact);
+    // }
```
Comment (Techvarun1991) [inline src/Components/AddNewMember.js:30]: Remove Unused Comments

File: src/Configurations/AuthenticationConfiguration.js [hunk 3]
```diff
@@ -9,7 +9,7 @@ export default class AuthenticationConfiguration
         return url2;
     }
 
-    static SignUp()
+    static signUp()
```
Comment (Techvarun1991) [inline src/Configurations/AuthenticationConfiguration.js:12]: Function naming should be in proper format

**Review by Techvarun1991 — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (anupkore) [reply src/App.js]: Spaces removed and code formatted according to the standards

File: src/Configurations/AuthenticationConfiguration.js (hunk 3 above)
Comment (anupkore) [reply src/Configurations/AuthenticationConfiguration.js:12]: > Function naming should be in proper format

Function Name Changed...All are in one format

File: src/App.js (hunk 1 above)
Comment (Techvarun1991) [reply src/App.js]: Noted..!!

File: src/Configurations/AuthenticationConfiguration.js (hunk 3 above)
Comment (Techvarun1991) [reply src/Configurations/AuthenticationConfiguration.js:12]: Changes Accepted!!

File: src/Components/AddNewMember.js (hunk 2 above)
Comment (anupkore) [reply src/Components/AddNewMember.js]: Doing it

Comment (Techvarun1991) [reply src/Components/AddNewMember.js]: okkk

Comment (anupkore) [reply src/Components/AddNewMember.js]: Resolved!!!

**Review by Techvarun1991 — APPROVED** [review]

**Pull request merged**